imagedumper_desktop.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 The Ebiten Authors
+// Copyright 2022 The Ebitengine Authors
```

Comment:
Don't change the year for the existing file.

Reply:
The original imagedumper_desktop was moved to imagedumper_notmobile, that diff just looks weird

Reply:
I see... In this case both file's year should not be changed.

Reply:
I moved everything back anyway